src/migration/1665020208581-addSubmissionProblemScores.ts

```diff
@@ -0,0 +1,68 @@
+import { MigrationInterface, QueryRunner } from 'typeorm'
+
+export class addSubmissionProblemScores1665020208581 implements MigrationInterface {
+  name = 'addSubmissionProblemScores1665020208581'
+
+  // prettier-ignore
+  public async up(queryRunner: QueryRunner): Promise<void> {
```

Comment:
There are so many constraints dropped and then re-added. Looking into it, my guess is that this the way it is and the SQL requires constraints to be added in a specific order which can't be satisfied until after the new table is added.